src/query/xasl.h

```diff
@@ -747,6 +748,9 @@ struct list_spec_node
 {
   REGU_VARIABLE_LIST list_regu_list_pred;	/* regu list for the predicate */
   REGU_VARIABLE_LIST list_regu_list_rest;	/* regu list for rest of attrs */
+  REGU_VARIABLE_LIST list_regu_list_build;	/* regu list for hash build */
+  REGU_VARIABLE_LIST list_regu_list_probe;	/* regu list for hash probe */
+  int hash_list_scan_yn;	/* Is hash list scan possible? */
```

Comment:
`hash_list_scan_yn` is declared as bool type in the `struct hash_list_scan`.

Reply:
I wrote int because bool type cannot be moved in XASL to stream